sencha-workspace/SlateDemonstrationsTeacher/app/controller/Viewport.js

```diff
@@ -1,45 +0,0 @@
-Ext.define('SlateDemonstrationsTeacher.controller.Viewport', {
-    extend: 'Ext.app.Controller',
-
-
-    // controller configuration
-    views: [
-        'Dashboard'
```

Comment:
@aliyeysides `Dashboard` view getting required went missing in this app too and was generating a warning in the console